### ISSUE TITLE
Bug: Adding a reference deletes rest of the line

### ISSUE THREAD

**Issue by atetzner** [theskumar/autolink-references-mkdocs-plugin#3]
Assuming the following initial markdown:
```
Lorem ipsum dolor sit amet, consetetur PROJ-1234 sadipscing elitr
```

My expectation is, that the plugin automatically transforms the above text to something like:
```
Lorem ipsum dolor sit amet, consetetur [PROJ-1234](https://jira.mycorp.com/browse/PROJ-1234) sadipscing elitr
```

But instead everything after the issue reference in the line is deleted:
```
Lorem ipsum dolor sit amet, consetetur [PROJ-1234](https://jira.mycorp.com/browse/PROJ-1234)
```

As far as I can tell, the [currently constructed regex](https://github.com/theskumar/autolink-references-mkdocs-plugin/blob/2ee314cadbea00cc74027949a465c33bb7f8bfed/autolink_references/main.py#L10-L11) is for example:
```
[\[]?PROJ-([0-9]+)[\]]?(\(?.*\)?)
```

I think that the part of the regex that should match the round brackets and the original markdown link is wrong as it matches not only the round braces and the link but instead everything in the same line after the reference. See [here](https://regex101.com/r/xF7k3A/1) for an explanation.

My suggestion is to change it to 
```
\[?PROJ-([0-9]+)\]?(\([^)]*\))?
```
as this will match only the reference and optionally the surrounding markdown link. See [here](https://regex101.com/r/PrNeJG/1) for an explanation.

<sub>Andreas Tetzner <[andreas.tetzner@daimler.com](mailto:andreas.tetzner@daimler.com)> on behalf of Daimler TSS GmbH. [Provider Information](https://github.com/Daimler/daimler-foss/blob/master/PROVIDER_INFORMATION.md).</sub>

**Comment by Andre601** [comment]
Can confirm this. I encounter the same issue, which is rather annoying.

**Comment by theskumar** [comment]
This should be resolved now. Thank for all the patience guys!

**Comment by atetzner** [comment]
Can confirm that it is fixed in 0.2.1

Thank you, @theskumar
